src/ValidatorFactory.php

```diff
@@ -53,11 +55,26 @@ public function loadTranslator() : Translator
         $loader = new FileLoader(new Filesystem(), $this->basePath . $this->namespace);
         $loader->addNamespace($this->namespace, $this->basePath . $this->namespace);
         $loader->load($this->lang, $this->group, $this->namespace);
-        return new Translator($loader, $this->lang);
+        return static::$translator = new Translator($loader, $this->lang);
     }
 
     public function __call($method, $args)
     {
         return call_user_func_array([$this->factory, $method], $args);
     }
+
+    /**
+     * Translate a key based on translation files
+     *
+     * @param string $key
+     * @return string
+     */
+    public static function trans(string $key): string
```

Comment:
I don't think this `trans()` function belongs to the current class 🤔

Actually, I don't believe that it is necessary because now with the implementation above you could just do `$validator->translator->trans($key)`, right?

Reply:
Yes you could do that but if you ever change the property to protected and not public then it would break.  I typically only expose functionality that my classes provide rather than making the dependencies public but if you wish I can make it work the way you wish.

Reply:
I will add some tests in the morning but it will require some code such as a custom rule object to be created.